docs/src/model_components/carbon-chemistry.md

```diff
@@ -28,9 +28,9 @@ This is sufficient when computing ``fCO_2`` at the surface, but if we wanted to
 carbon_chemistry(; DIC, Alk, T, S, P = 100)
 ```
 
-We may also be interested in the pH so we can request that be outputted:
+We may also be interested in the total pH so we can request that be outputted:
```

Comment:
It would be useful to add that "When output=Val(:variable) is specified, the named variable is outputted instead of fCO2."  The user might be expecting fCO2 _and_ the named variable to be outputted.